### PR TITLE
sfdisk: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
Instructions taken from the arch wiki page. There were a couple more. I haven't used this command so I'm being conservative on what I added to the page.

File: pages/linux/sfdisk.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# sfdisk
+
+> Display or manipulate a disk partition table.
+> More information: <https://manned.org/man/sfdisk>.
```
Comment (acuteenvy) [inline pages/linux/sfdisk.md:4]: This was changed for all pages in #13637.
```suggestion
> More information: <https://manned.org/sfdisk>.
```

File: pages/linux/sfdisk.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> Display or manipulate a disk partition table.
+> More information: <https://manned.org/sfdisk>.
+
+- Back up the partition layout to a file:
+
+`sudo sfdisk -d {{path/to/device}} > sda.dump`
+
+- Restore a partition layout:
+
+`sudo sfdisk /dev/sda < sda.dump`
```
Comment (spageektti) [inline pages/linux/sfdisk.md:12]:
```suggestion
`sudo sfdisk {{/dev/sdX}} < sda.dump`

- Set the type of a partition:

`sfdisk --part-type {{/dev/sdX}} {{partition_number}} {{swap}}`

- Delete a partition:

`sfdisk --delete {{/dev/sdX}} {{partition_number}}`

- Display [h]elp:

`sfdisk --help`
```

File: pages/linux/sfdisk.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# sfdisk
+
+> Display or manipulate a disk partition table.
+> More information: <https://manned.org/sfdisk>.
+
+- Back up the partition layout to a file:
+
+`sudo sfdisk -d {{path/to/device}} > sda.dump`
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/sfdisk.md:8]: Path to device has been used before and devices come in other name formats than sdX. I should rename the dump name though.

File: pages/linux/sfdisk.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> Display or manipulate a disk partition table.
+> More information: <https://manned.org/sfdisk>.
+
+- Back up the partition layout to a file:
+
+`sudo sfdisk {{-d|--dump}} {{path/to/device}} > {{path/to/file.dump}}`
+
+- Restore a partition layout:
+
+`sudo sfdisk {{path/to/device}} < {{path/to/file.dump}}`
```
Comment (spageektti) [inline pages/linux/sfdisk.md:12]:
```suggestion
`sudo sfdisk {{/dev/sdX}} < {{path/to/file.dump}}`

- Set the type of a partition:

`sfdisk --part-type {{/dev/sdX}} {{partition_number}} {{swap}}`

- Delete a partition:

`sfdisk --delete {{/dev/sdX}} {{partition_number}}`

- Display [h]help:

`sfdisk --help`
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**